airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3014,6 +3171,15 @@ components:
           description: Custom notes left by users for this Dag Run.
           type: string
 
+    DeleteQueueEvent:
+      type: object
+      properties:
+        cutoff_time:
+          description: The cutoff time of DatasetDagQueue.
+          format: date-time
+          type: string
+          nullable: true
+
```

Comment:
```suggestion
```

We don't need this any longer since we are using `Before` via querystring, right?

Reply:
Yep, just removed it.  Thanks!